auditlogger_test.go

```diff
@@ -0,0 +1,37 @@
+package ladon_test
+
+import (
+    "testing"
+
+    . "github.com/ory/ladon"
+    . "github.com/ory/ladon/manager/memory"
+    "github.com/stretchr/testify/assert"
+    "github.com/stretchr/testify/mock"
+)
+
+type MockAuditLogger struct {
```

Comment:
If we do [this](https://github.com/ory/ladon/pull/91#discussion_r150478852) you could remove this mock and replace it said struct and use a stub logger that implements `log.Logger` here instead.